MTFLOW_caller.py

```
@@ -300,74 +258,65 @@ def caller(self,
-                    grid_e_coeff = 0.7  # Adjust grid parameters to try and fix the grid, while also keeping the reduced number of streamwise_points
+                    # Adjust grid parameters to try and fix the grid, while also keeping the reduced number of streamwise_points
+                    grid_e_coeff = 0.7  
                     grid_x_coeff = 0.5
                     streamwise_points = 141
-                    if debug:
-                        logger.info(f"Trying suggested coefficients e={grid_e_coeff} and x={grid_x_coeff}")
-
-                # If the suggested coefficients do not work, we try a random number approach to try to brute-force a grid
+                elif check_count == 10: 
+                    exit_flag_gridtest = ExitFlag.CRASH  # If the grid is still incorrect after 10 tries, we assume that the grid is not fixable and exit the loop
+                    return exit_flag_gridtest, iter_count_gridtest
```

Ruff (0.8.2)

Undefined name `iter_count_gridtest`

(F821)